docs/RELEASING.md

```diff
@@ -0,0 +1,5 @@
+## Release Process
+
+- [Create a Jira release ticket](https://jira.sonarsource.com/projects/DS) like [this one](https://sonarsource.atlassian.net/browse/DS-200)
```

Comment:
obviously, we haven't created an example yet... but I think it would be valuable. Any ideas for alternatives?

Reply:
To avoid confusion, I'd make it an obvious placeholder. 
I think an example is the best, but maybe we can list the fields to be filled with a short explanation?